src/web/types.ts

```diff
@@ -39,9 +39,9 @@ declare global {
 }
 
 export type OpenAiProperties<
-  ToolInput extends UnknownObject = {},
+  ToolInput extends UnknownObject = Record<string, never>,
   ToolOutput extends UnknownObject = UnknownObject,
-  ToolResponseMetadata extends UnknownObject = UnknownObject,
+  ToolResponseMetadata extends UnknownObject = Record<string, never>,
```

Comment:
Why did you change this line?
```suggestion
  ToolResponseMetadata extends UnknownObject = UnknownObject,
```